tools/font-edit/twin-fedit.c

```diff
@@ -608,23 +620,29 @@ static void write_char(char_t *c)
     offset += 1;
 }
 
-int main(int argc, char **argv)
+static void print_offsets()
```

Comment:
The function name is confusing.

Reply:
Perhaps I need to first clarify the purpose of this function.

The function of `print_offset()` is to print the number of strings read for each drawn character from `nchar`.

For example, the drawn character `\0` include drawing operations 'm', 'l', and 'e' contain `28` strings that are read.

```
/* 0x0 '\0'  offset 0 */
    0, 24, 42, 0, 2, 4,
    0, 24, /* snap_x */
    -42, -21, -15, 0, /* snap_y */
    'm', 0, 0,
    'l', 0, -42,
    'l', 24, -42,
    'l', 24, 0,
    'l', 0, 0,
    'e',
```
Therefore, the offset for the next drawn string, `' '`, is 28, and `offsets[0x00]=28`. 
```
/* 0x20 ' '  offset 28 */
    0, 4, 0, 0, 2, 3,
    -128, 0, /* snap_x */
    -21, -15, 0, /* snap_y */
    'e',
```

Here are my alternative naming choices for this function:
1. `print_unicode_offset_values()`
2. `print_drawn_char_offsets()`
3. `print_unicode_offsets()`

Reply:
> Perhaps I need to first clarify the purpose of this function.
> The function of `print_offset()` is to print the number of strings read for each drawn character from `nchar`.

You can name it `generate_font_metrics`.

Refefence:
* [gcode-text](https://github.com/keith-packard/gcode-text)
* [HersheyView](https://github.com/wholder/HersheyView)
* [Intro to Font Metrics](https://westonthayer.com/writing/intro-to-font-metrics/)